bedrock/newsletter/views.py

```diff
@@ -663,7 +663,7 @@ def newsletter_subscribe(request):
                     errors.extend(form.errors[fieldname])
 
         # form error messages may contain unsanitized user input
-        errors = list(map(escape, errors))
+        errors = [escape(e) for e in errors]
```

Comment:
Not a comment: What/why this change?

Reply:
I think @pmac added this in his commit, but it seems like it's to make sure we're escaping error messages from the server correctly?

Reply:
I'm sure that was me. It's just more clear what is happening and it's supposed to be a bit quicker and more efficient. It's doing the same thing.